scripts/breakpad-taskcluster.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# TaskCluster build script for building Breakpad.
+# Submit a task like this to build:
+: <<'EOF'
+{
+	"created": "2016-04-09T00:49:49.410Z",
+	"deadline": "2016-04-09T01:49:49.410Z",
+	"provisionerId": "aws-provisioner-v1",
+	"workerType": "emulator-jb",
+	"retries": 0,
+	"expires": "2017-04-08T22:33:21.202Z",
+	"payload": {
+		"image": "taskcluster/desktop-build:0.1.11",
+		"command": [
+			"/bin/sh",
+			"-c",
+			"curl \"https://raw.githubusercontent.com/${SOCORRO_FORK:=mozilla/socorro}/${SOCORRO_BRANCH:=master}/scripts/breakpad-taskcluster.sh\" | bash"
+		],
+		"artifacts": {
+			"public/breakpad.tar.gz": {
+				"type": "file",
+				"path": "/home/worker/breakpad.tar.gz",
+				"expires": "2016-12-04T18:46:14.355Z"
```

Comment:
If we remove the value it will default to 1 year from the deadline date, which is probably good? Alternatively, consider hardcoding it into the far future.